adapters/AdColony/Public/Headers/GADMAdapterAdColonyExtras.h

```diff
@@ -17,4 +17,7 @@
 /// Enables reward dialogs to be shown after an advertisement.
 @property BOOL showPostPopup;
 
+/// Enables test ads for your application without changing dashboard settings.
+@property BOOL testMode;
```

Comment:
This extra is not being used by the adapter. I think you might missing one commit.

Reply:
You're right, thanks! I had a bad merge somewhere.